resolwe/flow/models/utils.py

```diff
@@ -321,13 +321,9 @@ def hydrate_input_references(input_, input_schema, hydrate_values=True):
                 output["__id"] = data.id
                 output["__type"] = data.process.type
                 output["__descriptor"] = data.descriptor
-                output["__entity_name"] = None
+                output["__entity_name"] = getattr(data.entity, 'name', None)
```

Comment:
How about supporting `data.entity_name()` and `data.collection_name()`? I assume we'll need this in several places.

Reply:
Currently, my counter is 2. I'll skip this for now, but when such statement stack up we should do it...